.github/workflows/ci.yml

```diff
@@ -54,6 +54,7 @@ jobs:
       - name: Install dependencies with typo3/cms-core:${{ matrix.typo3 }}
         run: |
           composer require typo3/cms-core:${{ matrix.typo3 }} --no-progress
+          composer require typo3/cms-extensionmanager:${{ matrix.typo3 }} --no-progress
```

Comment:
Wieso resolved? Die Zeile ist ja noch immer da.

Reply:
Die Zeile ist mit 98cde73 entfernt worden, das ist hier auf den alten Commit bezogen und wurde deshalb in der Ansicht nicht aktualisiert